DfE.FindInformationAcademiesTrusts/Pages/Trusts/_TrustLayout.cshtml

```diff
@@ -44,8 +47,11 @@
           </header>
           @RenderBody()
         </main>
-        <partial name="_SourceList"/>
-        <partial name="_ReportProblemLink"/>
+        @if (Model.ShowSourcesAndUpdates)
+        {
+          <partial name="_SourceList"/>
+          <partial name="_ReportProblemLink"/>
```

Comment:
This link was (originally) supposed to be present on every single page with data (i.e. all trust pages and the search results page). Does it definitely need removing for edit page?